python_modules/dagster/dagster_tests/storage_tests/test_io_manager.py

```diff
@@ -756,6 +756,27 @@ def handle_output(self, context, obj):
     assert result.success
 
 
+def test_asset_group_name():
```

Comment:
can you add a test for assets with no group name set?

Reply:
I've added the test asserting group is _default_, thanks.